flowcraft/generator/inspect.py

```diff
@@ -1530,10 +1530,19 @@ def _get_run_hash(self):
 
         # Get name of the pipeline from the log file
         with open(self.log_file) as fh:
-            header = fh.readline()
-
-        # Regex supports absolute paths and relative paths
-        pipeline_path = re.match(".*\s([/\w/]*\w*.nf).*", header).group(1)
+            # Searches for the first occurence of the nextflow pipeline
+            # file name in the .nextflow.log file
+            while 1:
+                line = fh.readline()
+                if not line:
```

Comment:
Does this happen when the file is empty? If so, add a short comment stating that. What happens then? if we don't get the `pipeline_path`?

Reply:
There is no way to catch this exception at the moment. But we can improve by adding an InspectError or ReportError to this, depending on the situation.